TMS.Net07.Homework3.CurrencyAndCalculator/TMS.Net07.Homework3.Low.Currency/TMS.Net07.Homework3.CurrencyAndCalculator/Program.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework3.CurrencyAndCalculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            const float BuyUsdToByn = 2.6250f;
+            const float SellUsdToByn = 2.6310f;
+            const float BuyUsdToEur = 0.8252f;
+            const float SellUsdToEur = 0.8315f;
+            const float BuyUsdToRub = 75.5051f;
+            const float SellUsdToRub = 76.9710f;
+
+            const float BuyEurToByn = 3.1650f;
+            const float SellEurToByn = 3.1680f;
+            const float BuyEurToRub = 91.4940f;
+            const float SellEurToRub = 94.6320f;
+
+            const float BuyBynToRub = 28.8243f;
+            const float SellBynToRub = 29.9132f;
+
+            while (true)
+            {
+                Console.Write("Input currency FROM: ");
+                string currencyFrom = Console.ReadLine().ToLower();
+
+                Console.Write("Input currency TO: ");
+                string currencyTo = Console.ReadLine().ToLower();
+
+                Console.Write("Choice BUY/SELL: ");
+                string buyOrSell = Console.ReadLine().ToLower();
+
+                Console.Write("Input amount: ");
+                bool isParseAmount = float.TryParse(Console.ReadLine(), out float currencyAmount);
```

Comment:
For money it's better to use `decimal` for higher accuracy.
Initially, `decimal` was invented for money

Reply:
Ok, I changed.